app/src/ui/app.tsx

```diff
@@ -473,18 +473,14 @@ export class App extends React.Component<IAppProps, IAppState> {
   }
 
   private getDotComAccount(): Account | null {
-    const state = this.props.appStore.getState()
-    const accounts = state.accounts
-    const dotComAccount = accounts.find(
+    const dotComAccount = this.state.accounts.find(
```

Comment:
These changes help ensure that we're all talking about the same instances of accounts during one render cycle.